src/main/java/seedu/address/model/FriendlyLink.java

```diff
@@ -208,7 +208,7 @@ public boolean hasPair(Pair pair) {
     }
 
     /**
-     * Adds a {@pair} to {@code FriendlyLink}.
+     * Adds a {code @pair} to {@code FriendlyLink}.
```

Comment:
There is a small typo here.
```suggestion
     * Adds a {@code pair} to {@code FriendlyLink}.
```

Reply:
Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87